open_dread_rando/files/lua_libraries/room_names.lua

```diff
@@ -0,0 +1,527 @@
+		collision_camera_012 = "Map Station Access",
+		collision_camera_013 = "Map Station",
+		collision_camera_014 = "EMMI Zone Exit West",
+		collision_camera_015 = "Early Ice Room",
+		collision_camera_016 = "EMMI Zone Exit Northwest",
+		collision_camera_017 = "Central Unit",
+		collision_camera_018 = "EMMI Zone Middle Path",
+		collision_camera_019 = "Central Unit Access",
+		collision_camera_020 = "EMMI Zone Exit Northeast",
+		collision_camera_021 = "Spin Boost Tower",
+		collision_camera_022 = "Save Station Center",
+		collision_camera_023 = "Chozo Warrior Arena",
+		collision_camera_024 = "Golzuna Tower",
+		collision_camera_025_B = "Total Recharge Station North",
+		collision_camera_026 = "Golzuna Arena",
+		collision_camera_027 = "Transport to Ferenia",
+		collision_camera_028 = "Map Station Access Secret",
+		collision_camera_029 = "Elun Transport Access",
+		collision_camera_030 = "Spin Boost Room",
+		collision_camera_031 = "Energy Recharge Station",
+		collision_camera_032 = "Pulse Radar Room",
+		collision_camera_033 = "Cross Bomb Tutorial",
+		collision_camera_034 = "Transport to Hanubia",
+		collision_camera_035 = "Spider Magnet Elevator",
+		collision_camera_036 = "Above Golzuna",
+		collision_camera_037 = "Transport to Elun",
+		collision_camera_038 = "Transport to Dairon",
+		collision_camera_039 = "Above Pulse Radar",
+		collision_camera_040 = "Save Station East"
+	}
+
+	local hanubia = {
+		collision_camera_000 = "Transport to Ferenia",
+		collision_camera_001 = "Transport to Ghavoran",
+		collision_camera_002 = "Ferenia Shortcut",
+		collision_camera_003 = "Navigation Station",
+		collision_camera_004 = "Entrance Tall Room",
+		collision_camera_005 = "Gold Chozo Warrior Arena",
+		collision_camera_006 = "Total Recharge Station North",
+		collision_camera_007 = "Transport to Itorash",
+		collision_camera_008 = "Ship Room",
+		collision_camera_009 = "Speedboost Puzzle Room",
+		collision_camera_010 = "Tank Room",
+		collision_camera_011 = "EMMI Zone Exit West",
+		collision_camera_012 = "Central Unit",
+		collision_camera_013 = "EMMI Zone Exit East",
+		collision_camera_014 = "Orange EMMI Introduction",
+		collision_camera_015 = "Total Recharge Station East",
+		collision_camera_016 = "Escape Room 3",
+		collision_camera_018 = "Escape Room 2",
+		collision_camera_019 = "Escape Room 1",
+		collision_camera_020 = "Raven Beak X Arena",
+		collision_camera_1000 = "collision_camera_1000 (H)"
+	}
+
+	local itorash = {
+		collision_camera_000 = "Save Station",
+		collision_camera_001 = "Transport to Hanubia",
+		collision_camera_002 = "Elevator to Raven Beak",
+		collision_camera_003 = "Elevator Cutscene",
+		collision_camera_004 = "Raven Beak Arena"
+	}
+
+	local full_dict = {
+		["Artaria"] = artaria,
+		["Burenia"] = burenia,
+		["Cataris"] = cataris,
+		["Dairon"] = dairon,
+		["Elun"] = elun,
+		["Ferenia"] = ferenia,
+		["Ghavoran"] = ghavoran,
+		["Hanubia"] = hanubia,
+		["Itorash"] = itorash
+	}
```

Comment:
these should be the internal IDs rather than readable names, e.g. `s010_cave` rather than `Artaria`. if you need the readable names for other purposes make another dict that maps IDs to names